docs/getstarted/getting-started.md

```diff
@@ -0,0 +1,277 @@
+
+    By default, when you change a setting, it applies to all your VS Code workspaces (**User Settings**). Switch to the **Workspace** tab to change a setting that only applies to the current workspace.
+
+    > **Tip**: Use the search box in the Settings Editor to quickly find a setting.
+
+There are many more ways to customize your VS Code experience, such as [changing default keyboard shortcuts](/docs/getstarted/keybindings.md), adding [code snippets](/docs/editor/userdefinedsnippets.md), or by adding extensions from the [Visual Studio Marketplace](#step-5-install-a-language-extension).
+
+## Step 4: Write some code
+
+VS Code is first and foremost a code editor, so let's start by writing some code! VS Code has built-in support for JavaScript, TypeScript, HTML, CSS, and more. In this tutorial, you create a sample JavaScript file and use some of the code editing features that VS Code offers.
+
+VS Code supports many programming languages and in the next step, you'll install a language extension to add support for a different language.
+
+1. In the Explorer view, create a new file `app.js`, and start typing the following JavaScript code:
+
+    ```javascript
+    function sayHello(name) {
+        console.log('Hello, ' + name);
+    }
+
+    sayHello('VS Code');
+    ```
+
+    As you type, you should see suggestions popping up that help you complete your code (*IntelliSense*). You can use the `kbstyle(Up)` and `kbstyle(Down)` keys to navigate the suggestions, and `kbstyle(Tab)` to insert the selected suggestion.
+
+    Notice also the formatting of the code (*syntax highlighting*), to help you distinguish between different parts of the code.
+
+    ![Screen capture that shows IntelliSense in action for a JavaScript file.](images/getting-started/javascript-intellisense.gif)
+
+1. Put the cursor on the `Hello,` string, select the lightbulb icon, and then select **Convert to template string** to use a Code Action to simplify the string concatenation.
+
+    **Code Actions** are suggestions that help you apply quick fixes to your code. You can also use the `kb(editor.action.triggerSuggest)` keyboard shortcut to open the lightbulb menu.
+
+    ![Screenshot that shows the lightbulb Code Action to convert a string concatenation to a template string.](images/getting-started/code-action-template-string.png)
+
+    You can further experiment with IntelliSense in the different files in your workspace. For example, in the `index.html` file, start typing `<` to see suggestions for HTML tags.
+
+    > **Tip**: Toggle the **File** > **Auto Save** menu item to automatically save files whenever you make changes.
+
+> **Tip**: You can further enhance your coding with artificial intelligence (AI) and [GitHub Copilot](/docs/copilot/overview.md). For example, get suggestions for lines of code or entire functions, perform code refactorings, generate code-related artifacts like tests, or produce code documentation, all from within the editor.
```

Comment:
Maybe this shouldn't be formatted as a tip note since: 1) there's another tip note directly above it, 2) we'd consider Copilot important enough to be top-level info.

Reply:
I wonder if Copilot should be its own top-level step at the very end? Considering how valuable / important we consider it. We could then also highlight ways it helps you get started with or configure your editor (i.e. `@vscode`).

Reply:
On it!